pkg/controller/registry/resolver/resolver.go

```diff
@@ -73,6 +73,8 @@ func (r *SatResolver) SolveOperators(namespaces []string, csvs []*v1alpha1.Clust
 			predicates = append(predicates, WithChannel(sub.Spec.Channel))
 		}
 
+		predicates = append(predicates, WithoutDeprecatedProperty())
```

Comment:
this applies for finding updates / new installs, but we also need this predicate when searching for dependency candidates - see line 302 in getBundleInstallables